templates/collections/blocks/promoted_pages.html

```diff
@@ -3,13 +3,13 @@
 <h2>{{ value.heading }}</h2>
 <p>{{ value.sub_heading }}</p>
 
-<ul class="row card-group">
+<ul class="row card-group--list-style-none">
```

Comment:
Similar to the comment above. We should avoid mixing BEM and Bootstrap on the same element.